src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/recordings/batch_help_full.txt

```diff
@@ -501,10 +497,14 @@ Global Arguments
                         examples.
     --verbose         : Increase logging verbosity. Use --debug for full debug logs.
 
-(env) C:\Code\Forks\azure-cli>cmd /c az batch application summary show --help 
+(env) D:\GitHub\azure-cli\src\command_modules\azure-cli-batch\azure\cli\command_modules\batch\tests>cmd /c az batch application summary show --help 
 
 Command
     az batch application summary show: Gets information about the specified application.
+        This operation returns only applications and versions that are available for use on compute
+        nodes; that is, that can be used in an application package reference. For administrator
+        information about applications and versions that are not yet available to compute nodes, use
+        the Azure portal or the Azure Resource Manager API.
```

Comment:
We could potentially overwrite this docstring to change the wording from "Azure Resource Manager API" to "use the az batch application list command".

Reply:
Individual parameter help text can be overridden in _params.py - though this isn't really major